selene_sdk/predict/model_predict.py

```diff
@@ -517,13 +555,15 @@ def get_predictions(self,
         -------
         None
             Writes the output to file(s) in `output_dir`. Filename will
-            match that specified in the filepath.
+            match that specified in the filepath. In addition, if any base
+            in the given or retrieved sequence, the row labels .txt file will
```

Comment:
`if any base in the given or retrieved sequence is unknown` something like that?